test/fixtures/misc-templates/deployment-w-progress.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: extensions/v1beta1
```

Comment:
Would the test/fixtures/long-running/undying-deployment.yml.erb fixture work for this? It just sleeps, and you can modify it to add the `progressDeadlineSeconds` by using the block form of `deploy_fixtures`.